resteasy-client-reactor-netty/src/test/java/org/jboss/resteasy/client/jaxrs/engines/ReactorNettyClientHttpEngineTest.java

```diff
@@ -6,8 +6,10 @@
 import io.netty.handler.codec.http.HttpResponseStatus;
 import io.netty.handler.codec.http.QueryStringDecoder;
 import io.netty.util.concurrent.DefaultEventExecutor;
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
I am surprised the build is green with `.*`, I believe Resteasy builds used to fail this style (or maybe allows it in test?).  Please remove `.*` to be consistent.

Reply:
The checkstyle is likely more lenient in tests, but I haven't checked.